README.md

```diff
@@ -1,2 +1,237 @@
-# booking-app-back-end
-Final Capstone Project to build a booking application using  ruby on rails as a back-end stack  
+<a name="readme-top"></a>
+
```

Comment:
You are missing reference to Documentation in README. Maybe add a section for Documentation

Reply:
This has been fixed, thank you